libs/pages/elewa/news/src/lib/components/news-section/news-section.component.html

```diff
@@ -0,0 +1,18 @@
+<div>
+    <div class="title">
+        <h1>News & Stories </h1>
+    </div>    
+    <div class="filter-section">
+        <elewa-group-button-plain [innerText]="'all-news'" (queryString)="filterSelector($event)" />
+        <elewa-group-button-plain [innerText]="'elewa'" (queryString)="filterSelector($event)" />
+        <elewa-group-button-plain [innerText]="'italanta'" (queryString)="filterSelector($event)" />
+        <elewa-group-button-plain [innerText]="'venture-labs'" (queryString)="filterSelector($event)" />
+        <elewa-group-button-plain [innerText]="'press'" (queryString)="filterSelector($event)" />
```

Comment:
You could use a loop on this, but better to just have the button local here instead of a whole reusable component